ArticleTemplates/audioTemplate.html

```diff
@@ -85,7 +88,24 @@ <h1 class="headline selectable card__title--audio"><span class="card__titletext"
     __COMMENTS_MODULE__
 
     <script type="text/javascript">
-        window.initTemplate();
+        GU.Bootstrap.init({
```

Comment:
Minor, but is there the concept of partials so you don't have to update this across all templates anytime you add or change anything?